notebooks/workshop.ipynb

```diff
@@ -73,7 +73,9 @@
     "\n",
     "    $ conda activate abis_gbi_course_cellpose_2023\n",
     "    \n",
-    "Before creating a new environment, remember to deactivate the current one:\n",
+    "Now run the scripts or notebooks as desired in this environment. Go to the additional instructions for notebooks below.\n",
+    "\n",
+    "(When finished working with one environment, before creating a new environment, remember to deactivate the current one:)\n",
```

Comment:
bracket and ``:`` seems strange
I think we should remove the brackets

Reply:
fixed in https://github.com/ome/ABIS-GBI-imaging-course-07-2023/pull/3/commits/1e3434e451cd91857d806331402fd1f965e8c6a8